core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/math/sum.kt

```diff
@@ -95,6 +97,11 @@ internal fun <T : Number> Iterable<T?>.sum(type: KType): T =
         else -> throw IllegalArgumentException("sum is not supported for $type")
     }
 
+// T: Number? -> T
```

Comment:
`//` -> `/** */ `?

Reply:
sure, it makes sense here to have them as KDoc